README.md

```diff
@@ -20,4 +28,14 @@ Provide a custom `values.yaml`:
 $ helm install metrics-server-app -f values.yaml
 ```
 
-Deployment to Tenant Clusters is handled by [app-operator](https://github.com/giantswarm/app-operator).
+ ## Release Process
+
+* Ensure CHANGELOG.md is up to date.
+* Create a new GitHub release which will push the tarball to the [default-catalog].
```

Comment:
Does it mean pushing tag?

Reply:
Yes for apps we want to create a GitHub release that links to the changelog.

Creating the release pushes the tag but I can make this more explicit? Since this is the trigger. WDYT?

Reply:
I didn't know that. Yeah so maybe dropping a sentence would help some lost souls like myself.

Reply:
Yes good point. Done in https://github.com/giantswarm/metrics-server-app/pull/7

I'm getting this aligned across the apps repos.